src/common/test/myProfileMocking.ts

```diff
@@ -155,3 +207,189 @@ export const getVerifiedData = (
   },
   ...overrides,
 });
+
+function createNewProfileNode(dataType: EditDataType): MultiItemProfileNode {
```

Comment:
There seems to be almost similar function defined in editData.ts line 227:

`export function createNewProfileNode<T extends MultiItemProfileNode>(...`

I wonder if we could apply it here some way?

Reply:
Good catch!

Reply:
Done.